### PR TITLE
Adbp 2662 fix bugs errors

### PR DESCRIPTION
**Pull request by catluong101digital**
fix bugs and errors

### CODE REVIEW

**Review by Toan101Digital — APPROVED** [review]

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 1]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
+            // justifyContent: "space-between",
+          }}
+          keyboardShouldPersistTaps="handled"
+        >
+          <View style={styles.containerStyle}>
+            <View style={styles.headerContainer}>
+              <TouchableOpacity style={styles.backButtonContainer} onPress={onPressBack}>
+                <ArrowRightBoldIcon size={18} color={colors.secondaryColor} />
+              </TouchableOpacity>
+              <Text style={{ color: colors.secondaryButtonLabelColor }}>
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:92]: inline style

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 2]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
-            currentBalanceValue={'0'}
-            isSelector={false}
-          />
-          <View style={styles.rangeWrapperStyle}>
-            <View style={styles.rangeInputContainerStyle}>
-              <Text style={styles.rangeInputTitleStyle}>
-                {i18n?.t('register_qr.setup_acc.lbl_daily_limit')}
+    <View style={{ flex: 1 }}>
+      <KeyboardAvoidingView
+        style={{ flex: 1 }}
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:74]: inline style

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 3]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
-            }}
-            isCurrentBalance={false}
-            currentBalanceValue={'0'}
-            isSelector={false}
-          />
-          <View style={styles.rangeWrapperStyle}>
-            <View style={styles.rangeInputContainerStyle}>
-              <Text style={styles.rangeInputTitleStyle}>
-                {i18n?.t('register_qr.setup_acc.lbl_daily_limit')}
+    <View style={{ flex: 1 }}>
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:72]: inline style

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 4]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
 const innerStyles = StyleSheet.create({
   inputContainer: {
-    flexDirection: "row",
-    alignItems: "center",
-    justifyContent: "center",
-    borderColor: "#1b1b1b",
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderColor: '#1b1b1b',
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:250]: use theme colors

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 5]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
+    width: '100%',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
     marginTop: 9,
   },
   errorText: {
     marginLeft: 10,
-    color: "#1b1b1b",
+    color: '#1b1b1b',
```
Comment (sharmal33) [inline src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:274]: use theme colors

File: src/components/qr-transfer-component/scan-to-pay-component/index.tsx [hunk 6]
```diff
@@ -106,17 +105,17 @@ const ScanToPayComponent: React.FC<ScanToPayComponentTypes> = (props: ScanToPayC
...
       <View style={styles.containerStyle}>
         <View style={innerStyles.cameraContainer}>
-          {!!isActivateScan && (
+          {
             <QRCodeScanner
               onRead={(e: any) => scannedQR(e)}
               flashMode={
                 isTorchOn ? RNCamera.Constants.FlashMode.on : RNCamera.Constants.FlashMode.off
               }
               cameraStyle={{ height: height - (height * 21) / 100, width: width }}
```
Comment (sharmal33) [inline src/components/qr-transfer-component/scan-to-pay-component/index.tsx:114]: inline style

File: src/components/transfer-details-component/qr-transfer-details-component.tsx [hunk 7]
```diff
@@ -252,13 +251,14 @@ const QrTransferDetailsComponent = forwardRef(
                     referenceNo:
                       confirmResult?.Data?.Initiation?.SupplementaryData?.PaymentServiceProviderExt
                         ?.PspReference,
-                    Description: 'Mobile Number',
+                    Description: 'DuitNow QR',
```
Comment (sharmal33) [inline src/components/transfer-details-component/qr-transfer-details-component.tsx:254]: get it from translation file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx [hunk 8]
```diff
@@ -61,200 +69,218 @@ const SetupQRAccountComponent: React.FC<SetupQRAccountComponentTypes> = (
...
+            </View>
+            <View style={styles.titleContainerStyle}>
+              <Text style={styles.titleStyle}>{i18n?.t('register_qr.setup_acc.lbl_title')}</Text>
+              <Text style={styles.subTitleStyle}>
+                {i18n?.t('register_qr.setup_acc.lbl_sub_title')}
+              </Text>
+            </View>
+            <Selector
+              title={'Account'}
+              value={maskString('1234569044')}
```
[the remark this answers is not in the document]
Comment (catluong101digital) [reply src/components/qr-transfer-component/qr-registration-component/setup-qr-account-component/index.tsx:105]: It was here before, let me check with Kayathri

**Pull request closed without merging**